src/main/java/cartPage/EstimateShippingAndTaxesTable.java

```diff
@@ -0,0 +1,13 @@
+package cartPage;
+
+import org.openqa.selenium.By;
+
+public class EstimateShippingAndTaxesTable {
+    public final String estimateShippingAndTaxesTable = "//*[@id='cart']/div/div[2]/div[2]";
+
+    public By countryDropdownLocator = By.xpath(estimateShippingAndTaxesTable + "//*[@id='estimate_country']");
+    public By stateDropdownLocator = By.xpath(estimateShippingAndTaxesTable + "//*[@id='estimate_country_zones']");
```

Comment:
It will be good to have some validation methods here?:)

Reply:
The same as above. I will add some methods there